### ISSUE TITLE
Update pod

### ISSUE THREAD

**Issue by olgagalchenko** [m-lab/ndt7-client-ios#81]
Now that this repo is using Locate V2 API, can we release a new pod version? 

@miguelangelnet, you've previously mentioned that you wanted to update the pod yourself, but I'm happy to open a PR for it if it would be helpful.

**Comment by stephen-soltesz** [comment]
While I've never done this before, is publishing the latest NDT7 pod self-service? https://cocoapods.org/pods/NDT7

* https://guides.cocoapods.org/making/using-pod-lib-create.html#deploying-your-library
* https://stackoverflow.com/questions/41361399/how-to-publish-new-official-pod-version

I'm wondering if the `pod repo push ...` asks the cocoapods service to fetch the latest config from the public repo, without needing credentials?

EDIT: not quite -- https://guides.cocoapods.org/making/getting-setup-with-trunk.html
There is a process to "Claiming an existing library" if necessary.

We should also update the README with publishing steps.

**Comment by miguelangelnet** [comment]
Hi @olgagalchenko , thank you for your help!,

I'm waiting for approval from my current company to be able to continue contributing to this open source library. While I'm waiting for this, I'm more than happy to give you the specific steps needed to be able to public the changes you did.

Please, open a new PR with this:

1. Verify README.md for the new version (update pod 'NDT7', '0.0.2' to pod 'NDT7', '0.0.3')
2. Update NDT7.podspec for new version (s.version = '0.0.3')
3. Create a PR for these changes, and, after merge the changes on Master, continue with the next steps:
4. Check last tags to know how add comment to the next tag you are going to create: 
    1. git tag
    2. git tag -l -n20
    3. git show 0.0.2
5.  Create a new tag release, and add the comments in the same way it was done before. Example: 
    1. git tag -a 0.0.3 -m "Releasing version 0.0.3
        1. Websocket update for download and upload.
        2. Upload test improvement.
        3. New MLap protocol updated.
        4. Example app updated.
        5. Documentation updated.
        6. README file updated."
5. push:
    1. git push origin 0.0.3
6. Go to GitHub and create a new release from tag - and mark pre-release if needed. Example:
    1. TITLE: NDT7 - Websocket refactor.
    2. Released on **Friday, May 21, 2019**. This is a beta release with a download and upload speed test beta version for NDT7 using new URL to get closer MLab server.
    3. ```diff
    4. - Warning: As beta version, there may be braking changes until 0.0.1 is released.
    5. ```
    6. 
    7. ### Added
    8. - New URL to get closer Mlab server.
    9. 
    10. ### Updated
    11. - README file.
7. Cocoapods release. Please, follow the instructions here: https://guides.cocoapods.org/making/getting-setup-with-trunk.html, and I can add the maintainers, so, others can add maintainers and update CocoaPods.

Thanks!

**Comment by critzo** [comment]
@miguelangelnet Thanks very much for these notes! 
I would suggest that @stephen-soltesz will create an M-Lab account to be used on CocoaPods as a co-maintainer. We can then do a release. At that point we can discuss a process fore new co-maintainers, and add others as there is interest and availability.

**Comment by stephen-soltesz** [comment]
```
$ pod trunk push --allow-warnings NDT7.podspec 
Updating spec repo `trunk`
Validating podspec
 -> NDT7 (0.0.3)
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | xcodebuild:  note: Planning build
    - NOTE  | xcodebuild:  note: Constructing build description
    - WARN  | xcodebuild:  NDT7/Sources/WebSocket.swift:426:32: warning: initialization of 'UnsafeMutablePointer<UInt8>' results in a dangling pointer
    - NOTE  | xcodebuild:  NDT7/Sources/WebSocket.swift:426:70: note: implicit argument conversion from '[UInt8]' to 'UnsafePointer<UInt8>' produces a pointer valid only for the duration of the call to 'init(mutating:)'
    - NOTE  | xcodebuild:  NDT7/Sources/WebSocket.swift:426:70: note: use the 'withUnsafeBufferPointer' method on Array in order to explicitly convert argument to buffer pointer valid for a defined scope
    - WARN  | xcodebuild:  NDT7/Sources/WebSocket.swift:1042:32: warning: initialization of 'UnsafePointer<UInt32>' results in a dangling pointer
    - NOTE  | xcodebuild:  NDT7/Sources/WebSocket.swift:1042:46: note: implicit argument conversion from '[UInt32]' to 'UnsafePointer<UInt32>' produces a pointer valid only for the duration of the call to 'init(_:)'
    - NOTE  | xcodebuild:  NDT7/Sources/WebSocket.swift:1042:46: note: use the 'withUnsafeBufferPointer' method on Array in order to explicitly convert argument to buffer pointer valid for a defined scope
    - NOTE  | xcodebuild:  warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the App editor. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  note: Using codesigning identity override: 

Updating spec repo `trunk`
```

@olgagalchenko can you confirm the new version WAI?

**Comment by olgagalchenko** [comment]
IT WORKS!!!!
![giphy](https://user-images.githubusercontent.com/915730/109885554-89ae2600-7c33-11eb-9d62-10095089b4a1.gif)
